cmds/portmaster-start/lock.go

```diff
@@ -32,14 +32,42 @@ func checkAndCreateInstanceLock(name string) (pid int32, err error) {
 		return 0, createInstanceLock(lockFilePath)
 	}
 
-	// check if process exists
+	// Check if process exists.
 	p, err := processInfo.NewProcess(int32(parsedPid))
-	if err == nil {
-		return p.Pid, nil
+	if err != nil {
+		// A process with the locked PID does not exist.
+		// This is expected, so we can continue normally.
+		return 0, createInstanceLock(lockFilePath)
+	}
```

Comment:
Use `errors.Is(err, processInfo.ErrorProcessNotRunning` as we might have a different error here as well. I.e. `PermissionDenied`.